ABIDE_I/data_convert.py

```diff
@@ -0,0 +1,40 @@
+"""ABIDE I phenotype data conversion from CSV to TSV format.
+
+This script finds the data file as downloaded from http://www.nitrc.org/frs/downloadlink.php/4912 in .csv format and
+converts it to .tsv format file to be in compliance with BIDS specification. The TSV files are deposited in the
+`phenotype/` directory along with the data dictionary in JSON format.
+"""
+
+import argparse
+from pathlib import Path
+
+import pandas as pd
+
+
+def get_args():
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, description=__doc__)
+    parser.add_argument('-i', '--input-file', type=Path, metavar='INPUT_FILE', required=True,
+                        help='Path to the phenotype file in CSV format as downloaded from ABIDE I webpage.')
+    parser.add_argument('-o', '--output-dir', type=Path, metavar='OUTPUT_DIR', required=False,
+                        default=Path(__file__).parent, help='Path to output BIDS formatted directory.')
```

Comment:
The input argument requires path to phenotype file as downloaded from ABIDE I webpage. 

For the output argument, I was not sure which would be better:
- the script outputs `phenotype.tsv` file in the directory path provided by the user OR
- the script outputs `phenotype/phenotype.tsv` in the directory provided by the user

Reply:
Additionally, we want the parent directory of the `dictionary.json` and `phenotype.tsv` files to be the same. There is a world in which a user could give two different output paths to `dictionary.py` and `data_convert.py` scripts. 
One not-fully-thought-out potential resolution would be to run `dictionary.py` and `data_convert.py` from a single script, let's say, `bidsify_phenotype.py`.

Reply:
I'm fine with either two scripts or one. As long as we make explicit that you _should_ keep the dictionary and data in the same directory, I think that's fine.

re: input arguments: So that we can be consistent with all other datasets I'd suggest using the arguments of `--input` and `--output` because sometimes the inputs are files and sometimes they are directories. I think `--output` should always be a directory, though.

Thought: if we keep two scripts, we want to make sure that the naming of the output filenames in both scripts matches, so that there is a clear .json/.tsv pair

Reply:
@Arshitha For the behavior of the `-i`/`--input` and `-o`/`--output` arguments I would like the expected behavior to be the same across datasets:

- The `--input` argument should take the path to the directory containing the exactly name as-expected/as-downloaded phenotype file. This way the user only needs to remember to always point to the directory containing the downloaded phenotype files, which will be more than one for most datasets.
- The `--output` argument should be the BIDS root or some other directory where the `data_convert.py` will produce a `phenotype/` subfolder and will name single filed datasets as `phenotype.{tsv,json}` under that `phenotype/` subfolder.

That said, I don't think either of the input or output arguments should include a file's base name at the end.

Next: I do like the idea of a CLI at the root of the repository called something simple like `bidsify.py` or `run.py` or `convert.py` that takes a minimum set of arguments and options. Usage something like:

```shell
# convert the ABCD data dictionaries to BIDS JSON
# support option name alternatives like --dictionary or just -j
python3 bidsify.py ABCD /path/to/input /path/to/output --json

# convert the ABIDE_I tabular data to BIDS TSV
# support option name alternatives like --data or --tabular or just -t
python3 bidsify.py ABIDE_I /path/to/input /path/to/output --tsv

# convert both the PNC data dictionary to BIDS JSON and the tabular data to BIDS TSV
# support option name alternatives like -b or --all or -a
python3 bidsify.py PNC /path/to/input /path/to/output --both
```

Lastly: I second @dmoracze 's BIG WARNING idea to notify the user in the top-level README and maybe also in the script's help message and maybe even in every command line call that "The whole output BIDS phenotype directory containing the data dictionary JSONs and data TSVs should be placed in the BIDS root-level directory" or some similar message.

Alright, enough of this big comment. Thoughts?

Reply:
All this sounds good, I'll refine this PR to include these changes. 

My only comment to `-i/--input` argument being a directory, and not a file, is to make an explicit note in the README of the expected filename/s, wherever necessary, and ask users to file an issue in case the as-downloaded filenames have been revised by dataset maintainers. I don't see this happening but this is just to cover all bases.

Reply:
I think we should do that AND do a try/except where we try to read the file as it should be named, but raise a clear exception message if the file is not there as expected to be placed and named.